pkg/sql/opt/xform/custom_funcs.go

```diff
@@ -1917,46 +1917,29 @@ func (c *CustomFuncs) GenerateGeoLookupJoins(
 		return
 	}
 
-	if !invertedidx.IsGeoIndexFunction(fn) {
-		panic(errors.AssertionFailedf(
-			"GenerateGeoLookupJoins called on a function that cannot be index-accelerated",
-		))
-	}
-
-	function := fn.(*memo.FunctionExpr)
 	inputProps := input.Relational()
+	function := fn.(*memo.FunctionExpr)
 
-	// Extract the the variable inputs to the geospatial function.
-	if function.Args.ChildCount() < 2 {
-		panic(errors.AssertionFailedf(
-			"all index-accelerated geospatial functions should have at least two arguments",
-		))
-	}
-
-	// The first argument should come from the input.
-	variable, ok := function.Args.Child(0).(*memo.VariableExpr)
-	if !ok {
-		panic(errors.AssertionFailedf(
-			"GenerateGeoLookupJoins called on function containing non-variable inputs",
```

Comment:
Why is it safe for these panics to be removed? Were they mostly debugs and now things are probably fine?

Reply:
They haven't actually been removed, they've just been moved inside `TryGetInvertedJoinCondFromGeoFunc`.